src/loaders/zarr_utils/utils.ts

```diff
@@ -119,7 +119,7 @@ function compareZarrArraySize(
   }
 }
 
-const EPSILON = 0.0000001;
```

Comment:
Just curious: why shrink this?

Reply:
It's not a great reason, and I can undo it, but it's because Matheus showed me this data
```
SEG
"scale": [
                                5.003550305,
                                1.0,
                                0.5349045238095238,
                                0.270825,
                                0.270825

RAW
scale": [
                                5.003550305,
                                1.0,
                                0.53,
                                0.2708333333333333,
                                0.2708333333333333
```

Reply:
:shrug: works for me. If our arbitrary fudge factor to reflect real-world conditions doesn't match real-wold conditions, there's no rigorous reason not to change it.